### PR TITLE
Android examples

### PR DESCRIPTION
**Pull request by lasote**
With https://github.com/conan-io/examples2/pull/35

### CODE REVIEW

File: examples/cross_build/android/ndk.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Cross building to Android with the NDK
+======================================
+
+In this example, we are going to see how to cross-build a Conan package to Android.
+
+First of all, download the Android NDK from `the download page <https://developer.android.com/ndk/downloads>`_
+and unzip it. In MacOS you can also install it with ``brew install android-ndk``.
+
+Then `go to the ``profiles`` folder in the conan config home directory (check it running :command:`conan config home`)
```
Comment (czoido) [inline examples/cross_build/android/ndk.rst:12]:
```suggestion
Then go to the ``profiles`` folder in the conan config home directory (check it running :command:`conan config home`)
```

File: examples/cross_build/android/ndk.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Both the library and the ``test_package`` executable are built for Android, so we cannot use them in our local computer.
+
+Unless you have access to a `root` Android device, running the test application or using the built library is not possible
+directly so it is more common to build an Android application that uses the ``hello`` library.
+
+Read more
+---------
+
+Check the example :ref:`Integrating Conan in Android Studio<examples_cross_build_android_studio>` to know how to use your
```
Comment (czoido) [inline examples/cross_build/android/ndk.rst:73]: We can link the tutorial: https://docs.conan.io/en/2.0/tutorial/consuming_packages/cross_building_with_conan.html and also link this in the tutorial.

File: examples/cross_build/android/android_studio.rst [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+   target_link_libraries(myconanapp ${log-lib} ZLIB::ZLIB)
+
+
+Building the application
+------------------------
+
+If we build our project we can see that `conan install` is called multiple times building the different configurations
+of ``zlib``.
+
+Then if we run the application in a Virtual Device or in a real device pairing ir with the QR code we can see:
```
Comment (czoido) [inline examples/cross_build/android/android_studio.rst:247]:
```suggestion
Then if we run the application in a Virtual Device or in a real device pairing it with the QR code we can see:
```

**Review by czoido — APPROVED** [review]

**Comment by czoido** [comment]
Really nice!

**Review by memsharded — APPROVED** [review]

**Review by franramirez688 — APPROVED** [review]

**Pull request merged**